great_expectations/checkpoint/v1_checkpoint.py

```diff
@@ -150,7 +150,7 @@ def run(
             run_id=run_id,
             run_results=run_results,
             checkpoint_config=self,
-            validation_result_url=None,  # TODO: Need to plumb from actions
+            validation_result_urls=[result.result_url for result in run_results.values()],
```

Comment:
Rather than a single URL like we support in the v0.18 checkpoint, I'm opting to have a collection. These are ordered in the same way that the actual results are but let me know if that isn't super clear or poses usability concerns

Reply:
is this field meant to be used directly by the end user? I wonder if a more idiomatic place to put the url would be inside the `validation_results` key in `CheckpointResult.describe()`.

Reply:
Hmm I quite like that actually!